File: Examples/CSharp/Text/CreateTitleMsStyle.cs
using System;
using System.Globalization;

namespace Aspose.Note.Examples.CSharp.Text
{
    public class CreateTitleMsStyle
    {
        public static void Run()
        {
            // ExStart:CreateTitleMsStyle
            // ExFor:Page
            // ExFor:Page.Title
            // ExFor:RichText
            // ExFor:RichText.ParagraphStyle
            // ExFor:RichText.Text
            // ExFor:Title
            // ExFor:Title.TitleText
            // ExFor:Title.TitleDate
            // ExFor:Title.TitleTime
            // ExSummary:Shows how to set a title for a page.

            string dataDir = RunExamples.GetDataDir_Text();
            string outputPath = dataDir + "CreateTitleMsStyle_out.one";

            var doc = new Document();
            var page = new Page();

            page.Title = new Title()
            {
                TitleText = new RichText()
                {
                    Text = "Title text.",
                    ParagraphStyle = ParagraphStyle.Default
                },
                TitleDate = new RichText()
                {
                    Text = new DateTime(2011, 11, 11).ToString("D", CultureInfo.InvariantCulture),
                    ParagraphStyle = ParagraphStyle.Default
                },
                TitleTime = new RichText()
                {
                    Text = "12:34",
                    ParagraphStyle = ParagraphStyle.Default
                }
            };

            doc.AppendChildLast(page);

            doc.Save(outputPath);

            // ExEnd:CreateTitleMsStyle

            Console.WriteLine("\nPage title setup successfully in microsoft OneNote style .\nFile saved at " + outputPath);
        }
    }
}

File: Examples/VisualBasic/Text/CreateTitleMsStyle.vb
Imports System.Globalization
Imports Aspose.Note

Namespace Text
    Public Class CreateTitleMsStyle
        Public Shared Sub Run()
            ' ExStart:CreateTitleMsStyle
            Dim dataDir As String = RunExamples.GetDataDir_Text()
            Dim outputPath As String = dataDir & Convert.ToString("CreateTitleMsStyle_out_.one")

            Dim doc = New Document()
            Dim page = New Page(doc)

            page.Title = New Title(doc) With {
                 .TitleText = New RichText(doc) With {
                     .Text = "Title text.",
                     .DefaultStyle = TextStyle.DefaultMsOneNoteTitleTextStyle
                },
                 .TitleDate = New RichText(doc) With {
                     .Text = New DateTime(2011, 11, 11).ToString("D", CultureInfo.InvariantCulture),
                     .DefaultStyle = TextStyle.DefaultMsOneNoteTitleDateStyle
                },
                 .TitleTime = New RichText(doc) With {
                     .Text = "12:34",
                     .DefaultStyle = TextStyle.DefaultMsOneNoteTitleTimeStyle
                }
            }

            doc.AppendChild(page)

            doc.Save(outputPath)
            ' ExEnd:CreateTitleMsStyle
            Console.WriteLine(Convert.ToString(vbLf & "Page title setup successfully in microsoft OneNote style ." & vbLf & "File saved at ") & outputPath)
        End Sub
    End Class
End Namespace
